srv/modules/runners/push.py

```diff
@@ -72,6 +72,16 @@ def proposal(filename = "/srv/pillar/ceph/proposals/policy.cfg", dryrun = False)
     pillar_data.output(common)
     return True
 
+def __create_dirs(path, root):
```

Comment:
since we have the exact same method here and in `populate.py:757` we might consider to create a helper lib.